README.md

```diff
@@ -129,7 +129,7 @@
 https://aj193.github.io/Capstone-project/index.html
 
 ## 🚀 Loom Link <a name="Loom-Link"></a>
-https://www.loom.com/share/24614a0619f2470b8388d46accd262bf
+https://www.loom.com/share/84413da408af4a7c81803071a289cd0e
```

Comment:
- [ ] As stated before by the previous reviewer, kindly consider presenting a piece of code in your project by sharing your IDE and highlighting what you are proud of it as stated in the requirements in the following image. Kindly take this chance as a practice for interview skills because when doing a test project for a job application. In the interview, you will ask about your code and why you implemented with this way
>![image](https://user-images.githubusercontent.com/53402600/233059186-7b4a2084-25d4-40b7-b674-755247598861.png)

Reply:
@omarsalem7 
hello reviewer kindly guide me correctly that what should I do to show that

Reply:
Hi @AJ193,
You should make a presentation video containing all the following guidelines:

- [ ] recorded a video presenting the features of the project.
- [ ] The maximum length of the video is 5 minutes.
- [ ] The video contains a clear and objective description of the project in professional English
- [ ] The video contains a demo of the project, showing the pages in different screen sizes.
- [ ] **The video contains a highlight about a piece of the code which made you proud of it**